site/source/docs/javascript_usage.md

```diff
@@ -43,6 +43,30 @@ EmbarkJS.onReady((error) => {
 });
 ```
 
+## Using `dappConnection`
+
+After reading the section above on `EmbarkJS.onReady`, you might be wondering, but where does EmbarkJS connect?
+
+The answer is: you decide! You can configure it using `dappConnection`, a config property you can find in the Contracts config (by default at `config/contracts.js`).
+
+You will notice that property name reused in some other config files too, like the Storage one, because you can configure where those connect too.
+
+`dappConnection` is a bit weird, but is quite simple. It's an array of strings where EmbarkJS will try each of those in order (from 0 to N) and as soon as one of the connections work, it will stop.
```

Comment:
> `dappConnection` is a bit weird, but is quite simple.

^ Haha that sounds like "we know we messed it up, but actually it's not that bad after all."

How about

> The expressions used within `dappConnection` come with special semantics. Here's how they work:

Reply:
Yeah, you're right haha